doc/ai-subnet.md

```diff
@@ -369,6 +369,54 @@ The first and **recommended method** is to use the `ethOrchAddr` flag to set the
 
 After you have completed these steps, you can start your _Mainnet AI Subnet_ Orchestrator using the binary or Docker image.
 
+#### Allow your AI Subnet Orchestrator to be Discovered
+
+To ensure that _AI Gateway_ nodes can discover your _AI Subnet_ Orchestrator, you need to write your service URI to the blockchain. This URI should be in the format `https://<PUBLIC_ORCH_IP_OR_URL>:<PUBLIC_ORCH_PORT>`. To write your service URI to the blockchain, go to the machine that contains your _Mainnet AI Subnet_ Orchestrator wallet and follow these steps:
+
+1. Install the [Foundry](https://book.getfoundry.sh/getting-started/installation) smart contract development toolchain:
+
+    ```bash
+    curl -L https://foundry.paradigm.xyz | bash
+    source /home/<USERNAME>/.bashrc
+    foundryup
+    ```
+
+2. Connect your _Mainnet Transcoding Network_ Orchestrator wallet to the Foundry cast using the following command:
```

Comment:
I think an easier flow could be to use the `--keystore` and `--password` flags supported by [cast send](https://book.getfoundry.sh/reference/cli/cast/send?highlight=keystore#cast-send) since Os use keystore files to run go-livepeer already. 

Ex. to call `setServiceURI()`

```
cast send --keystore <PATH_TO_KEYSTORE_FILE> --password <PASSWORD> ... "setServiceURI(string)" ...
```

This would also avoid the need to directly deal with private keys since the user only needs to deal with keystore files.